src/azure-cli/azure/cli/command_modules/advisor/_params.py

```diff
@@ -52,3 +52,5 @@ def load_arguments(self, _):
                    help='Exclude from recommendation generation.')
         c.argument('include', options_list=['--include', '-i'], action='store_true',
                    help='Include in recommendation generation.', validator=validate_include_or_exclude)
+        c.argument('configuration_name', options_list=['--configuration-name'],
```

Comment:
Unncessary to use `options_list` if you don't want to add abbrevation for the argument.

Reply:
OK, I've removed it.